infra/helper.py

```diff
@@ -457,15 +457,22 @@ def build_fuzzers_impl(project_name, clean, engine, sanitizer, architecture,
       ['docker', 'run', '--rm', '-i', '--cap-add', 'SYS_PTRACE'] +
       _env_to_docker_args(env))
   if source_path:
-    workdir = _workdir_from_dockerfile(project_name)
-    if workdir == '/src':
-      print('Cannot use local checkout with "WORKDIR /src".', file=sys.stderr)
-      return 1
+    if not mount_location:
+      workdir = _workdir_from_dockerfile(project_name)
+      if workdir == '/src':
+        print('Cannot use local checkout with "WORKDIR /src".', file=sys.stderr)
```

Comment:
nit: Add colon after WORKDIR and use string formatting rather than hardcoding /src.